test/PSRP.Tests.ps1

```diff
@@ -29,23 +29,44 @@ Describe " PowerShell Remoting basic functional tests" -Tag @("CI") {
             Get-PSSession|Remove-PSSession
         }
 
-        It "Remoting from Windows/Linux/MacOS to Linux with basic authentication with winrm/omicli should work" {
+        It "Remoting from Windows/Linux/MacOS to Linux with basic authentication with omicli over https should work" {
```

Comment:
This is omicli, which is a tool, not PowerShell or WSMan, we already have PowerShell and WSMan case for this, so it is different case with PowerShell and WSMan.
I my mind, 
PowerShell case use new-possession on windows and Linux.
WSMan case use winrm.cmd on Windows.
omicli case is a testing tool on Linux.
Do you agree with this?